examples/clogsample/CMakeLists.txt

```diff
@@ -1,9 +1,6 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-# Add clog subdirectory inc folder to create CLOG target
-add_subdirectory(submodules/clog/inc)
```

Comment:
This is because we built clog already in the CI? I think it might be good to support both inline build of clog (keeping this) and external package model. Maybe a config flag to control the inclusion of this directory, instead of just removing it? Then we actually have two test modes: one like you have here, and one where it builds everything with cmake?

Reply:
I very much dislike having special cases for different modes. Devs can easily add the output path of clog to their PATH, which will do the same thing. I technically could do that in CI, but installing the global tool is easy too.

Reply:
Ok. Makes sense.